.github/workflows/meson.yml

```diff
@@ -155,7 +182,7 @@ jobs:
       - name: configure
         run: |
           muon version
-          muon setup -Dtest=true build
+          muon setup -Dtest=enabled build
```

Comment:
noteworthy: like shown here, the option changes affect the build config interface, which should be properly announcement in an eventual release text. But with how the commits are named and not mentioning this in their desription this will now likely be forgotten
Also, can you elaborate on why it was changed?

Reply:
> A couple of these have actual dependency interactions, and it didn't make sense for some of the utils to be features while others were booleans.